interpreter/cling/lib/Interpreter/CIFactory.cpp

```diff
@@ -1333,7 +1335,7 @@ static void stringifyPreprocSetting(PreprocessorOptions& PPOpts,
     // Everything else points to them.
     DiagnosticOptions& DiagOpts = InvocationPtr->getDiagnosticOpts();
     llvm::IntrusiveRefCntPtr<DiagnosticsEngine> Diags =
-        SetupDiagnostics(DiagOpts);
+        SetupDiagnostics(DiagOpts, COpts.CUDADevice ? "cling-ptx" : "cling");
```

Comment:
I bet that this change is causing the test failures due to the output being different than expected. I am okay with showing `cling` as part of the diagnostic even for ROOT, but we will need to update the .ref files for the test suite. Can you take care of that, @SimeonEhrig or shall I?

Reply:
You are right. I thought I implemented it in such a way that it doesn't affect the output when CUDA mode is not enabled to avoid this kind of problems with the CI, but that's wrong.

@Axel-Naumann 
Can you please change the `.ref` files. I don`t have time to get involved at the moment. And please link the changes in this issue. Maybe then I understand how it works and can do it myself next time.